ovirt-vmconsole.spec.in

```diff
@@ -10,7 +10,7 @@ Summary:	oVirt VM console
 Name:		@PACKAGE_NAME@
 Version:	@PACKAGE_RPM_VERSION@
 Release:	@PACKAGE_RPM_RELEASE@%{?release_suffix}%{?dist}
-License:	GPLv3
```

Comment:
It is there since the beginning but headers in the source files say GPL 2+ and COPYING file is LGPL 2. I couldn't find anything about GPL 3 except here. To make it less messy, I think we should put GPL 2 or later here.

Reply:
Ok, done